static/src/javascripts/projects/commercial/modules/third-party-tags/outbrain-codes-new.js

```diff
@@ -0,0 +1,76 @@
+// @flow
+
+/*
+  Implements outbrain codes according to the new structure defined
+  here: https://trello.com/c/r1TDqVKP
+*/
+
+const normalizedTypes = {
+    defaults: 'defaults',
+    merchandising: 'nonCompliant',
+    nonCompliant: 'nonCompliant',
+};
+
+const outbrainCodes = {
+    defaults: {
+        desktop: { code: 'AR_12' },
+        tablet: { code: 'MB_6' },
+        mobile: { code: 'MB_4' },
+    },
+    epic: {
+        desktop: { code: 'AR_13' },
+        tablet: { code: 'MB_7' },
+        mobile: { code: 'MB_5' },
+    },
+    nonCompliant: {
+        desktop: { code: 'AR_14' },
+        tablet: { code: 'MB_8' },
+        mobile: { code: 'MB_10' },
+    },
+};
+
+const outbrainCode = function(
+    paramSet: string,
+    breakpoint: string
+): { code?: string, image?: string, text?: string } {
```

Comment:
`{ code?: string, image?: string, text?: string }`

looks like image and string are never included?

Since we otherwise throw, the return type of the function can be:

```js
{ code: string }
```

Reply:
I've put that in the cleanup story.